src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssDeclarations = sourceString.split(';');
```

Comment:
Splitting with split(';') works fine, but if the CSS string contains extra semicolons or empty lines, those will need to be processed as well.